src/main/java/org/fogbowcloud/saps/engine/core/database/Catalog.java

```diff
@@ -4,97 +4,55 @@
 import java.sql.Timestamp;
 import java.util.Date;
 import java.util.List;
-import java.util.Map;
-
 import org.fogbowcloud.saps.engine.core.dispatcher.notifier.Ward;
 import org.fogbowcloud.saps.engine.core.model.SapsImage;
 import org.fogbowcloud.saps.engine.core.model.SapsUser;
 import org.fogbowcloud.saps.engine.core.model.enums.ImageTaskState;
 
 public interface Catalog {
 
-	public String NONE = "None";
-	public int UNLIMITED = -1;
+	String NONE = "None";
```

Comment:
it seems no one is using this NONE variable :)

Reply:
Variable NONE removed